### ISSUE TITLE
EOL issues 

### ISSUE THREAD

**Issue by Machine-Maker** [eamodio/vscode-gitlens#883]
- GitLens Version: 10.1.1
  
- VSCode Version: 1.39.2
- OS Version: Windows 10 (10.0.18362 Build 18362)

Steps to Reproduce:

1. run `git config --global core.autocrlf true`

2. stage some changes

3. gitlens shows changes on the working tree but running `git status` doesn't show those changes. Looking at the file comparison, there is nothing hightlighting it so it has to be the eol characters.

Basically gitlens is showing that there are stages to be changed but `git status` says otherwise.
![image](https://user-images.githubusercontent.com/15055071/67429975-1d7ad700-f596-11e9-8cc7-e836af318f13.png)

**Comment by eamodio** [comment]
Do you see the same in the built-in vscode git?

**Comment by eamodio** [comment]
I am closing this issue because it needs more information and has not had recent activity.  Please re-open this issue when more details can be provided.

Thanks!

**Comment by Machine-Maker** [comment]
I figured out what the issue was. The inconsistency was between git on WSL and git on normal windows, because WSL always resets the file permissions. I ran git config core.filemode false to fix it.
